### PR TITLE
The dashboard of the mitre module is the same with or without a pinned agent

### LINKED ISSUE #6877: The dashboard of the mitre module is the same with or without a pinned agent
**Description**

### Unpinned agent
The dashboard of the mitre module is the same with or without a pinned agent

image

### Pinned agent
image

### PR DESCRIPTION
**Pull request by chantal-kelm**
### Description

This PR fixes the MITRE ATT&CK dashboard did not change the definition when pinning/unpinning the agent.

#### Changes
- Set donut type visualization

#### Side changes
- Set donut type visualization (GitHub and File Integrity monitoring)
 
### Issues Resolved

#6877 

### Test
[Provide instructions to test this PR]

### Check List
- [x] All tests pass
  - [x] `yarn test:jest`
- [ ] New functionality includes testing.
- [ ] New functionality has been documented.
- [x] Update [CHANGELOG.md](./../CHANGELOG.md)
- [ ] Commits are signed per the DCO using --signoff

### CODE REVIEW

**Comment by yenienserrano** [comment]
test: 🟡 

image
image

It seems to me that we were changing the pie chart to donut on all dashboards to unify visualizations.

As in threat hunting:

4.9.0 threat hunting
image

4.8.1 threat hunting
image

**Review by yenienserrano — COMMENTED** [review]
https://github.com/wazuh/wazuh-dashboard-plugins/pull/6878#issuecomment-2256551496

File: plugins/main/server/integration-files/visualizations/agents/agents-mitre.ts
Comment (Desvelao) [inline plugins/main/server/integration-files/visualizations/agents/agents-mitre.ts:1]: **nitpick**: I do not know why this file was added, the dashboard is now defined on the frontend side. This file is unused.

**Comment by Desvelao** [comment]
# Changes
- Set donut type visualization

# Side
- Set donut type visualization (GitHub and File Integrity monitoring)

File: plugins/main/server/integration-files/visualizations/agents/agents-mitre.ts
Comment (Desvelao) [reply plugins/main/server/integration-files/visualizations/agents/agents-mitre.ts:1]: Removed on https://github.com/wazuh/wazuh-dashboard-plugins/pull/6878/commits/0423f438e11cc78f2879cc9c1a8133bc23f34efd.

**Comment by Desvelao** [comment]
:green_circle: # Test

image
image

**Review by Desvelao — APPROVED** [review]
LGTM!

**Review by Desvelao — APPROVED** [review]
LGTM!

**Review by yenienserrano — DISMISSED** [review]
LGTM

image

**Review by yenienserrano — APPROVED** [review]
LGTM

image

image

**Review by Desvelao — APPROVED** [review]
LGTM!

**Pull request merged**